grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcService.java

```diff
@@ -62,17 +63,37 @@ static GrpcServiceBuilder builder() {
     boolean isFramed();
 
     /**
-     * Returns the {@link ServerServiceDefinition}s serviced by this service.
+     * Returns the {@link ServerServiceDefinition}s served by this service.
      */
     List<ServerServiceDefinition> services();
 
+    /**
+     * Returns the {@link ServerMethodDefinition} of the current request.
+     */
+    @Nullable
+    default ServerMethodDefinition<?, ?> methodDefinition(ServiceRequestContext ctx) {
```

Comment:
Should `HttpJsonTranscodingService` override this method as well?

Reply:
Oops, I forgot about it. 😓